libs/ledger-live-common/src/wallet-api/Exchange/server.ts

```diff
@@ -309,6 +317,7 @@ export const handlers = ({
         }),
       );
     }),
+    "custom.device.get": customWrapper<void, DeviceGetRequest>(() => deviceGet()),
```

Comment:
You could implement `deviceGet` in here directly IMO, if you still want to get the device from LLD and LLM you can receive it as a param like `accounts` or `manifest` instead of using a uiHook